docs/docusaurus/docs/core/configure_project_settings/access_secrets_managers/_aws_secrets_manager.md

```diff
@@ -0,0 +1,67 @@
+import GxData from '../../_core_components/_data.jsx'
+import PreReqFileDataContext from '../../_core_components/prerequisites/_file_data_context.md'
+
+### Prerequisites
+
+- An AWS Secrets Manager instance.  See [AWS Secrets Manager](https://docs.aws.amazon.com/secretsmanager/latest/userguide/tutorials_basic.html).
+- The ability to install Python packages with `pip`.
+- <PreReqFileDataContext/>.
+
+### Procedure
+
+1. Set up AWS Secrets Manager support.
+   
+   To use the AWS Secrets Manager with {GxData.product_name} you will first need to install the `great_expectations` Python package with the `aws_secrets` requirement.  To do this, run the following command:
+
+   ```bash title="Terminal"
+   pip install 'great_expectations[aws_secrets]'
+   ```
+
+2. Reference AWS Secrets Manager variables in `config_variables.yml`.
+
+   By default, `config_variables.yml` is located at: 'gx/uncomitted/config_variables.yml' in your File Data Context.
+
+   Values in `config_variables.yml` that start with `secret|arn:aws:secretsmanager` will be substituted with corresponding values from the AWS Secrets Manager.  However, if the keywords following `secret|arn:aws:secretsmanager` do not correspond to keywords in AWS Secrets Manager no substitution will occur.
+
+   You can reference other stored credentials within the keywords by wrapping their corresponding variable in `${` and `}`.  When multiple references are present in a value, the secrets manager substitution takes place after all other substitutions have occurred.
+
+   An entire connection string can be referenced from the secrets manager:
+
+   ```yaml title="config_variables.yml"
+    my_aws_creds:  secret|arn:aws:secretsmanager:${AWS_REGION}:${ACCOUNT_ID}:secret:dev_db_credentials|connection_string
```

Comment:
Do the `:` or `|` have any special meaning after `secret|arn:aws:secretsmanager`? Is the key in the secret store really `:${AWS_REGION}:${ACCOUNT_ID}:secret:dev_db_credentials|connection_string` once `${AWS_REGION}` and `${ACCOUNT_ID}` are resolved? It seems strang that it would start with a `:`.

Reply:
Unfortunately, I don't have access to an AWS secrets manager account myself so most of these examples are pulled from previous versions of the docs that were written by someone else.  I've been hesitant to change them specifically because I haven't been able to go into a secrets manager account to verify anything I may alter.